__DEFINES/setup.dm

```diff
@@ -1802,3 +1802,22 @@ var/list/weekend_days = list("Friday", "Saturday", "Sunday")
 #define SPORTINESS_NONE 1
 #define SPORTINESS_SUGAR 1.2
 #define SPORTINESS_SPORTS_DRINK 5
+
+//Breakable items
+#define BREAKABLE_ALL 15			//Item can break by being hit, or when used to hit something as a melee or a thrown weapon.
+#define BREAKABLE_HIT	3			//Item can break by being hit with either an unarmed attack or a weapon.
+#define BREAKABLE_UNARMED 1			//Item can break by being hit with an unarmed attack.
+#define BREAKABLE_WEAPON 2			//Item can break by being hit with a weapon. This includes projectiles.
+#define BREAKABLE_AS_ALL 12			//Item can break when it's used as a melee weapon, or when it ballistically collides with something.
```

Comment:
You can organize these macros better:
- Move them to their own file under `__DEFINES/`
- `BREAKABLE_ALL` could just be defined as `#define BREAKABLE_ALL ALL`, I think. So that adding and removing flags in the future doesn't require updating `BREAKABLE_ALL` as well.
- The flags could be ordered from smallest to biggest
- Compound flags such as `BREAKABLE_AS_ALL` could be written by combining previous flags (and therefore necessarily declared after them) like: `#define BREAKABLE_AS_ALL (BREAKABLE_WEAPON | BREAKABLE_AS_THROWN)` and so on.
- The values could be expressed in the form of bitshifts, so that you get a nice list of ordered contiguous integers like this:
```
#define BREAKABLE_UNARMED	(1<<0)
#define BREAKABLE_WEAPON	(1<<1)
```
and so on.

Reply:
> * `BREAKABLE_ALL` could just be defined as `#define BREAKABLE_ALL ALL`, I think. So that adding and removing flags in the future doesn't require updating `BREAKABLE_ALL` as well.
Where does the ALL come from/what does it mean?

Reply:
It's defined here https://github.com/vgstation-coders/vgstation13/blob/5c9b1517d08ac851ab37b0f6b13127c230149c26/__DEFINES/setup.dm#L212
It means every bitflag is set.

Reply:
Is that equivalent to ...11111111? I'm not sure that it'd work in this case because there is a flag BREAKABLE_NOMOB that overrides other flags.

Reply:
Should that be a thing? Why do it like that instead of a flag like BREAKABLE_MOB that works like the others?

Reply:
That makes sense, I changed it.